lib/ace/ext/papeeria/katex-demo.coffee

```diff
@@ -0,0 +1,31 @@
+define (require, exports, module) ->
```

Comment:
ah, damn. That's exactly what I meant "looks like a syntax error". Yes, please always add braces around function arguments.